### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2017-12-20

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#447]
## Time

**UTC Wed 20-Dec-2017 17:00 (05:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 20-Dec-2017 09:00 (09:00 AM) |
| US / Mountain | Wed 20-Dec-2017 10:00 (10:00 AM) |
| US / Central  | Wed 20-Dec-2017 11:00 (11:00 AM) |
| US / Eastern  | Wed 20-Dec-2017 12:00 (12:00 PM) |
| London        | Wed 20-Dec-2017 17:00 (05:00 PM) |
| Amsterdam     | Wed 20-Dec-2017 18:00 (06:00 PM) |
| Moscow        | Wed 20-Dec-2017 20:00 (08:00 PM) |
| Chennai       | Wed 20-Dec-2017 22:30 (10:30 PM) |
| Hangzhou      | Thu 21-Dec-2017 01:00 (01:00 AM) |
| Tokyo         | Thu 21-Dec-2017 02:00 (02:00 AM) |
| Sydney        | Thu 21-Dec-2017 04:00 (04:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2017-12-20&iso=20171220T17
* or http://www.wolframalpha.com/input/?i=05PM+UTC%2C+Dec+20%2C+2017+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1q35I8ONS2Aqsjhcci6OJ6vDdak5VngFcChhbmE8aCTg/edit>

## Agenda

Extracted from **TSC-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* build: update minimum kernel version to 3.10.0 [#14795](https://github.com/nodejs/node/pull/14795)

### nodejs/TSC

* Proposed New TSC Meeting Schedule [#444](https://github.com/nodejs/TSC/issues/444)
* Mentorship initiative ramping up. [#443](https://github.com/nodejs/TSC/issues/443)
* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
* Document guidelines for accountability expectations for all members of the TSC and CommComm [#311](https://github.com/nodejs/TSC/issues/311)
* Not all members of the GitHub org are members of the members team [#284](https://github.com/nodejs/TSC/issues/284)

## Invited

* Anna Henningsen @addaleax (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Daniel Bevenius @danbev (TSC)
* Evan Lucas @evanlucas (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* Josh Gavant @joshgav (TSC)
* Fedor Indutny @indutny (TSC)
* James M Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Brian White @mscdex (TSC)
* Myles Borins @MylesBorins (TSC)
* Ali Ijaz Sheikh @ofrobots (TSC)
* Rod Vagg @rvagg (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)
* Trevor Norris @trevnorris (TSC)
* Rich Trott @Trott (TSC)

## Notes

The agenda comes from issues labelled with `TSC-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Uberconference; participants should have the link & numbers, contact me if you don't.

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that

**Comment by fhinkel** [comment]
If we have #431 on the agenda, can we please invite @bnb

**Comment by Trott** [comment]
@fhinkel We agreed at the last meeting @jasnell attended that we would keep that labeled `tsc-agenda` but not pick it up until @jasnell is back from vacation, so that would be either January 3 or January 10. (Pretty sure he's back on January 3 but that meeting is currently slated for a time slot that is inconvenient for @jasnell. On the other hand, we do have a proposal for changing the meeting times, so we'll see.)

Related: I'm removing that item from this agenda and from the Google doc.  That said, if we think it needs discussing this week, please add it back or (perhaps slightly better) open a separate issue which more precisely frames the issue that needs discussion sooner rather than later.

**Comment by Trott** [comment]
Observers to invite: @gibfahn and @BridgeAR for https://github.com/nodejs/node/pull/14795

At the meeting, I will probably propose that we table https://github.com/nodejs/TSC/issues/443 because @mhdawson is on vacation this week and next.

@MylesBorins Do we need to keep https://github.com/nodejs/TSC/issues/311 on the agenda? (Not advocating, just asking.)

**Comment by rvagg** [comment]
`tsc-review` items (FYI this can be easily generated by https://github.com/nodejs/node-meeting-agenda, with `node-meeting-agenda tsc-review`):

### nodejs/build

* doc: note that release config is removed after 6m [#997](https://github.com/nodejs/build/pull/997)

### nodejs/node

* util: improve error message of _errnoException [#17626](https://github.com/nodejs/node/pull/17626)
* child_process: ignore undef/proto values of env [#15089](https://github.com/nodejs/node/pull/15089)
* Regression: Resolution algorithm collision [#14990](https://github.com/nodejs/node/issues/14990)
* build: update minimum kernel version to 3.10.0 [#14795](https://github.com/nodejs/node/pull/14795)
* module: remove unnecessary nonInternalExists check [#14664](https://github.com/nodejs/node/pull/14664)
* configure: fix type of arm_version [#14595](https://github.com/nodejs/node/pull/14595)
* Buffer.isEncoding regards an empty string as a valid encoding [#9654](https://github.com/nodejs/node/issues/9654)

### nodejs/TSC

* Approval to move node-heapdump into foundation [#257](https://github.com/nodejs/TSC/issues/257)

**Comment by Trott** [comment]
Looks like @danbev hasn't been added to the invite list yet, so @-mentioning them here for notification purposes.

**Comment by Trott** [comment]
(Also, just added @danbev manually to the original post here and the minutes doc.)

**Comment by danbev** [comment]
> (Also, just added @danbev manually to the original post here and the minutes doc.)

Thanks, I actually thought that this was because I marked that time in the spreadsheet as a bad time for me. l might not make it I'm afraid (still trying to see if I might get a sitter and will know for sure tomorrow).

**Comment by danbev** [comment]
Sorry, I won't be able to make it today.

**Comment by addaleax** [comment]
I won’t be able to make it either

**Comment by rvagg** [comment]
4am for me, opting out

**Comment by evanlucas** [comment]
Eating lunch with my little girl for her birthday, so I won't be able to make it today. I should be back next week though.

**Comment by BridgeAR** [comment]
Thanks for inviting me but I probably can not make it as I am going to be at the airport at that time. I guess @gibfahn is the best person to talk to in this case anyway.

**Comment by Fishrock123** [comment]
I'll be able to make and stream it

**Comment by dshaw** [comment]
Thanks, @rvagg. I need to start using https://github.com/nodejs/node-meeting-agenda, with `node-meeting-agenda neuf-agenda` for @nodejs/user-feedback.

**Comment by Trott** [comment]
Minutes: https://github.com/nodejs/TSC/pull/451

**Comment by joyeecheung** [comment]
Had a fever last night and was not able to make it, sorry.
